cfgov/v1/models/molecules.py

```diff
@@ -31,6 +33,34 @@ class ImageText5050(blocks.StructBlock):
     link_url = blocks.URLBlock(required=False)
     link_text = blocks.CharBlock(max_length=100, required=False)
 
+    def clean(self, data):
+        error_dict = {}
+        try:
+            values = super(ImageText5050, self).clean(data)
+        except ValidationError as e:
+            error_dict.update(e.params)
+            values = data
```

Comment:
@kave doesn't this return tuples? I thought it had to be converted to a `dict`. I might be remembering wrong.